src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/SolAboutApplicant1.java

```diff
@@ -26,7 +26,7 @@ public void addTo(final PageBuilder pageBuilder) {
                 .mandatoryWithLabel(Applicant::getLastName,
                     "${labelContentApplicantsOrApplicant1s} last name")
                 .mandatoryWithLabel(Applicant::getNameDifferentToMarriageCertificate,
-                    "Is ${labelContentApplicantsOrApplicant1s} name different to that on the"
+                    "Is ${labelContentApplicantsOrApplicant1sLC} name different to that on the"
```

Comment:
Did this line need to have "the" after Is: "Is the..."?

Reply:
will check with Michael I have sent him the screenshots 
<img width="627" alt="image" src="https://user-images.githubusercontent.com/13840402/145843943-b04ccdb0-1ec7-4917-9c92-53c3d7a7e4f5.png">

Reply:
That looks good to me actually 👍.